### ISSUE TITLE
Uncaught Error: ENOENT: no such file or directory, scandir 'C:\projects\temp\Muffin.Pictures.Arch...

### ISSUE THREAD

**Issue by koenmetsu** [fsprojects/atom-fsharp#93]
[Enter steps to reproduce below:]

1. Project with ProjectScaffold (sources are at https://github.com/koenmetsu/muffin.pictures.archiver)
2. Run target "RunTests" via atom's "Fake: Build"
3. "Fake: show builds" -> Select previously run build

The folder "RunTests - 09\07" doesn't exist.

**Atom Version**: 1.0.2
**System**: Microsoft Windows 7 Enterprise 
**Thrown From**: [atom-fsharp](https://github.com/fsprojects/atom-fsharp) package, v0.7.2

### Stack Trace

Uncaught Error: ENOENT: no such file or directory, scandir 'C:\projects\temp\Muffin.Pictures.Archiver\RunTests - 09\07'

```
At C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\node_modules\q\q.js:128

Error: ENOENT: no such file or directory, scandir 'C:\projects\temp\Muffin.Pictures.Archiver\RunTests - 09\07'
  at Error (native)
  at Object.fs.readdirSync (fs.js:805:18)
  at Object.fs.readdirSync (ATOM_SHELL_ASAR.js:441:28)
  at findFsProj (C:\Users\exi342\.atom\packages\atom-fsharp\lib\fsharp.js:2964:25)
  at Parser__parseProjectForEditor$ (C:\Users\exi342\.atom\packages\atom-fsharp\lib\fsharp.js:3005:13)
  at C:\Users\exi342\.atom\packages\atom-fsharp\lib\fsharp.js:2919:7
  at Emitter.module.exports.Emitter.emit (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\node_modules\event-kit\lib\emitter.js:82:11)
  at C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\src\pane-container.js:343:34
  at Emitter.module.exports.Emitter.emit (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\node_modules\event-kit\lib\emitter.js:82:11)
  at Pane.module.exports.Pane.setActiveItem (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\src\pane.js:238:22)
  at Pane.module.exports.Pane.addItem (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\src\pane.js:338:14)
  at Pane.module.exports.Pane.activateItem (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\src\pane.js:301:14)
  at C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\src\workspace.js:479:16
  at _fulfilled (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\node_modules\q\q.js:794:54)
  at self.promiseDispatch.done (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\node_modules\q\q.js:823:30)
  at Promise.promise.promiseDispatch (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\node_modules\q\q.js:756:13)
  at C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\node_modules\q\q.js:564:44
  at flush (C:\Users\exi342\AppData\Local\atom\app-1.0.2\resources\app.asar\node_modules\q\q.js:110:17)
  at process._tickCallback (node.js:357:13)

```

### Commands

```
  5x -3:15.4.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -3:14.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -2:04.6.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -2:01.2.0 editor:select-to-first-character-of-line (atom-text-editor.editor.mini.is-focused)
     -1:59.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:59.5.0 symbols-view:go-to-declaration (atom-text-editor.editor)
     -0:37.3.0 command-palette:toggle (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:33.9.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:33.9.0 FAKE:Build (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
 18x -0:33.2.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:29.4.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -0:29.4.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -0:28.6.0 command-palette:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-zenburn.theme-one-dark-ui)
     -0:26.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:26.7.0 FAKE:Show-Builds (atom-workspace.workspace.scrollbars-visible-always.theme-zenburn.theme-one-dark-ui)
     -0:26 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "excludeVcsIgnoredPaths": false,
    "themes": [
      "one-dark-ui",
      "zenburn"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-fsharp, v0.7.2
atom-yeoman, v0.3.11
file-icons, v1.5.8
file-type-icons, v0.7.0
language-aspx, v0.4.0
language-cshtml, v0.1.1
language-fsharp, v0.8.4
language-powershell, v2.0.1
linter, v1.2.3
minimap, v4.11.0
omnisharp-atom, v0.17.3
pigments, v0.8.3
project-manager, v1.15.11
project-sidebar, v1.3.0
trailing-spaces, v0.3.2
tree-view-git-status, v0.1.1
zenburn, v3.0.0

# Dev
No dev packages
```

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in 0.8.2

**Comment by koenmetsu** [comment]
using 0.8.4, not fixed here. What's with the assumption of the existence of a RunTests directory? I'm not seeing that folder on my filesystem either.

**Comment by Krzysztof-Cieslak** [comment]
It's not assuming existing of RunTest directory. It creates virtual in memory file with path PROJECT_PATH \ TARGET_NAME - CURRENT_DATE_TIME. Because Your default date format contains "\" it's treating it not as file name but as directory + file name.

Anyway, should be fixed in `0.9.1`
